pkgs/games/cgoban/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, writers
+, adoptopenjdk-jre-bin
+, fetchurl
+, makeWrapper
+}:
+stdenv.mkDerivation rec {
+  pname = "cgoban";
+  version = "3.5.23";
+
+  nativeBuildInputs = [ adoptopenjdk-jre-bin makeWrapper ];
+
+  src = fetchurl {
+    url = "http://files.gokgs.com/javaBin/cgoban.jar";
```

Do we have a versioned download URL available? If not please upload it to web.archive.org.

---

It does not. It does exist already on `web.archive.org`, and I had wayback recrawl the file (http://web.archive.org/web/20210717230228/https://files.gokgs.com/javaBin/cgoban.jar). How do I find the programmatic URL for downloading it?

---

```suggestion
    url = "http://web.archive.org/web/20210717230228/https://files.gokgs.com/javaBin/cgoban.jar";
```

you just use this URL. There is no real way to automatically update it.